packages/foam-vscode/src/features/wikilink-diagnostics.spec.ts

```diff
@@ -103,7 +103,7 @@ describe('Wikilink diagnostics', () => {
     expect(items[0].severity).toEqual(vscode.DiagnosticSeverity.Warning);
     expect(
       items[0].relatedInformation.map(info => info.location.uri.path)
-    ).toEqual([fileA.uri.path, fileB.uri.path]);
+    ).toEqual([fileB.uri.path, fileA.uri.path]);
```

Comment:
why this change between `fileA` and `fileB`?

Reply:
Because in the specification it's now `listByIdentifier` that takes care of returning its results in a stable order, instead of its user `find`. I thought that it would be better to enforce this behavior from the source itself, but if you don't agree I can move it downwards again. Anyway, that's the reason why the order was different, this happens in like 135 of `wikilinks-diagnostics.ts`:

```ts
const targets = workspace.listByIdentifier(target);
```